{{cpp/error/nested_exception/title|nested_ptr}}
{{cpp/error/nested_exception/navbar}}
{{ddcla|since=c++11|constexpr=c++26|1=
std::exception_ptr nested_ptr() const noexcept;
}}

返回指向存储的异常的指针，若存在。

===参数===
（无）

===返回值===
若存储异常则为指向它的 {{tt|std::exception_ptr}}，否则为默认构造的 {{tt|std::exception_ptr}}。

{{langlinks|en|ja|ru}}